src/settings/groups/GroupDetailsModel.ts

```diff
@@ -291,7 +328,7 @@ export class GroupDetailsModel {
 
 			if (isUpdateForTypeRef(GroupInfoTypeRef, update) && operation === OperationType.UPDATE) {
 				const updatedUserGroupInfo = await this.entityClient.load(GroupInfoTypeRef, this.groupInfo._id)
-				if (isSameId(this.groupInfo._id, [neverNull(instanceListId), instanceId])) {
+				if (isSameId(this.groupInfo._id, [assertNotNull(instanceListId, "got groupInfo update without instanceListId"), instanceId])) {
```

Comment:
do we really need these messages? This can only happen if something goes Really Wrong